hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/HoodieTimelineArchiver.java

```diff
@@ -459,6 +459,9 @@ private Stream<HoodieInstant> getCommitInstantsToArchive() {
 
   private Stream<HoodieInstant> getInstantsToArchive() {
     Stream<HoodieInstant> instants = Stream.concat(getCleanInstantsToArchive(), getCommitInstantsToArchive());
+    if (config.isMetastoreEnabled()) {
+      return Stream.empty();
```

Comment:
Why is `isMetastoreEnabled` turned on and `Stream.empty()` is returned here?

Reply:
For hudi metastore, timeline no more need to be archived

Reply:
For metastore implementation, is it possible to have something new like `MetastoreHoodieTiimelineArchiver` implement?

Reply:
> MetastoreHoodieTiimelineArchiver

This is a good idea and I think it can be implemented in steps.